backends/avx/ceed-avx-basis.c

```diff
@@ -25,14 +25,15 @@
 
 // Blocked Tensor Contact
 static inline int CeedTensorContract_Avx_Blocked(Ceed ceed, CeedInt A,
-                                          CeedInt B, CeedInt C, CeedInt J,
-                                          const CeedScalar *restrict t,
-                                          CeedTransposeMode tmode,
-                                          const CeedInt Add,
-                                          const CeedScalar *restrict u,
-                                          CeedScalar *restrict v,
-                                          const CeedInt JJ,
-                                          const CeedInt CC) {
+    CeedInt B, CeedInt C,
```

Comment:
This change seems inconsistent with the style below - see also this inconsistency rampant in 'include/ceed.h'